lib/synapse_pay_rest/api/users.rb

```diff
@@ -75,7 +75,10 @@ def create(payload:)
     def refresh(user_id:, payload:)
       path = "/oauth/#{user_id}"
       response = client.post(path, payload)
+      puts '-- Refresh -----------' if @logging
```

Comment:
Let's make these `Rails.logger` calls. Since this is our patched version of the gem, we can expect it to always be available in our app.